### PR TITLE
move notification resources to base module

### PR DESCRIPTION
**Pull request by Zayankovsky**
### Description
Moved notification resources to `:libnavigation-base`, which is necessary for implementing a notification interceptor in Android Auto (#5778). Also it might be useful for those, who would like to implement their custom `TripNotification`.

### CODE REVIEW

**Review by kmadsen — APPROVED** [review]
LGTM

File: libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/trip/notification/NotificationTurnIconResources.kt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-package com.mapbox.navigation.trip.notification.internal
+package com.mapbox.navigation.base.internal.trip.notification
 
 import androidx.annotation.DrawableRes
+import com.mapbox.navigation.base.R
 import com.mapbox.navigation.base.maneuver.model.BaseTurnIconResources
-import com.mapbox.navigation.trip.notification.R
 
 class NotificationTurnIconResources private constructor(
```
Comment (kmadsen) [inline libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/trip/notification/NotificationTurnIconResources.kt:7]: Maybe we should expose a resource api similar to [`MapboxLaneIconsApi`](https://github.com/mapbox/mapbox-navigation-android/blob/623705630c6b2cf20e16db82bbd56c5264bfdde4/libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/api/MapboxLaneIconsApi.kt#L15-L36)

We did this for android auto as well https://github.com/mapbox/mapbox-navigation-android/pull/4729

**Review by LukasPaczos — APPROVED** [review]

File: libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/trip/notification/NotificationTurnIconResources.kt (hunk 1 above)
Comment (Zayankovsky) [reply libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/trip/notification/NotificationTurnIconResources.kt:7]: `TurnIconHelper` looks like such an API. We can think of making it public.

**Comment by abhishek1508** [comment]
@Zayankovsky I think we missed to update the transifex and reflect that the strings were moved to a different path. Could you take care of that?

/cc @kmadsen @LukasPaczos

**Pull request merged**